setup-wizard.yml

```diff
@@ -119,14 +119,13 @@ fields:
       on your router and exposed to the Internet.
       You can also use the (dyn)DNS name of your node as `host`.
       The `host` part must be an external public IP address or DNS name of this node.
-      The recommended default TCP `port` is 9091.
+      The recommended default TCP `port` is 9091, but any port in the range 9091-9099 is usable.
```

Comment:
Should we check the valid range also in the regex?

Reply:
Done.